sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -338,6 +338,13 @@ export class OfferingValidationModel {
   /**
    * Actual tuition costs.
    */
+  @ValidateIf(
+    (offering: OfferingValidationModel) =>
+      !!offering.actualTuitionCosts &&
+      !!offering.programRelatedCosts &&
+      !!offering.mandatoryFees &&
+      !!offering.exceptionalExpenses,
```

Comment:
Would it still work if three values together exceed the maximum and the fourth value is provided as 0?

Reply:
If a field is left blank, then as per the ticket, it should not show a warning. If all the fields are filled in (even with zero) and one or more of the values exceeds the maximum, then it will show the warning.